Documentation/ceph-cluster-crd.md

```diff
@@ -1390,6 +1391,21 @@ spec:
     #externalMgrEndpoints:
       #- ip: 192.168.39.182
     #externalMgrPrometheusPort: 9283
+    # prometheus rule alerts values for overriding default prometheus rules values
+    # optional: specific alerts can be disabled by setting disabled field to true 
+    #alertRuleOverrides:
+    #  CephNodeDown:
+    #    disabled: true
+    #  CephMgrIsAbsent:
+    #    for: 1m
+    #    severityLevel: custom-severityLevel
```

Comment:
Do these need to be from a specific set such as warning, error, critical? Or can custom levels be defined somehow?

Reply:
I wasn't familiar with the ceph severity_level UI requirment explained [here](https://github.com/ceph/ceph-mixins/issues/54#issuecomment-488950655).

Now, if there is custom severity label that is not one of the accepted levels:  [warning, error, critical], prometheus template will be failed.

We can decide that instead of failing the templating in that case, we can just use the default value instead of the wrong custom value and log the reason for using the default value.

Please let me know which way you prefer to implement.